modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of AWS Load Balancer Controller
+
+After installing the operator, you can create an instance of `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator.only reconciles the resource with the name `cluster`.
```

Comment:
```suggestion
After installing the operator, you can create an instance of `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator will only reconciles the resource with the name `cluster`.
```

Reply:
We avoid using future tense in the docs, so IMO we should keep the present tense